### PR TITLE
new felinid/taj meow/purr sounds, 5 second emote cooldown reverted (mostly)

### PR DESCRIPTION
**Pull request by Neocloudy**
## About The Pull Request
- **the 5 second cooldown per emote is reverted for most emotes** - the only sound cooldown in this pr is the 2 second cooldown for all emotes, so this restores the original behavior before some recent mirrors
  - some emotes still have it (purr, wurble, bork, cackle, growl, clap, cough, whistle)
- felinid/taj meow/purr emotes now use the tg meow/purr sounds
  - no the oranges meow Cannot happen with this one
image
## How This Contributes To The Nova Sector Roleplay Experience
I feel like the 5 second cooldown wasn't really needed here in particular, and it's just really inconvenient :sob:

Also, the TG meow/purr sounds are a lot better than what we have. Both of our sounds for these were just annoying
## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>

https://streamable.com/q6dtpd

</details>

## Changelog
:cl:
del: [NOVA] the 5 second cooldown per emote has been mostly reverted: now only the 2 second cooldown is applied. these emotes still have cooldowns: purr, bork, wurble, cackle, growl, clap, cough, whistle
sound: the felinid/taj meow and purr emotes now use the tg meow and purr sounds
/:cl:

### CODE REVIEW

File: modular_nova/master_files/code/datums/emotes.dm [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /datum/emote
+	specific_emote_audio_cooldown = 0 SECONDS
```
Comment (GoldenAlpharex) [inline modular_nova/master_files/code/datums/emotes.dm:2]: I'm not against it being more adjusted to each sound, but having it be flat-out 0 means that people can *easily* spam them in order to create a LOT of noise spam for no reason. It's not an *if* it's going to happen, it *will* happen, there should still be some delay between the noises, and there should be a duration by default, for all of the people that might forget about it when adding their own sound emotes.

**Review by GoldenAlpharex — DISMISSED** [review]

File: modular_nova/master_files/code/datums/emotes.dm (hunk 1 above)
Comment (Neocloudy) [reply modular_nova/master_files/code/datums/emotes.dm:2]: setting it to 0 seconds means only the two second cooldown for all successful emote sounds gets applied, so this is the original behavior before the recent mirrors

anyway- what do you think it should be set to?

**Review by vinylspiders — CHANGES_REQUESTED** [review]

**Review by vinylspiders — CHANGES_REQUESTED** [review]
Also, you'll now have to also override `specific_emote_audio_cooldown = 5 SECONDS` for the following TG emotes: 
`/datum/emote/living/carbon/clap`, 
`/datum/emote/living/sneeze`, 
`/datum/emote/living/cough/`, and
`/datum/emote/living/carbon/whistle` 

plus the (modular) `/datum/emote/living/clap`

because they relied on the default being `5 SECONDS`.

**Review by vinylspiders — APPROVED** [review]

**Comment by Sadboysuss** [comment]
Removing the code that prevents audible emote spam on an hrp server so you can meow more often

**Comment by Hlscientist** [comment]
The new meows are a bit too silent... could you make them a bit louder?

**Pull request merged**